Apps/CocoaPods-FCM/src/AppDelegate.swift

```diff
@@ -23,65 +23,45 @@ class AppDelegate: NSObject, UIApplicationDelegate {
         // Initialize the Customer.io SDK
         CustomerIO.initialize(siteId: siteId, apiKey: apiKey, region: .US) { config in
             // Modify properties in the config object to configure the Customer.io SDK.
+            config.autoTrackPushEvents = true
             // config.logLevel = .debug // Uncomment this line to enable debug logging.
 
             // This line of code is internal to Customer.io for testing purposes. Do not add this code to your app.
             appSetSettings?.configureCioSdk(config: &config)
         }
+        // Initialize messaging features after initializing Customer.io SDK
         MessagingInApp.initialize(eventListener: self)
-
-        // Now that the Firebase and Customer.io SDK's are initialized, follow the rest of the required steps for the FCM push setup.
-        UNUserNotificationCenter.current().delegate = self
-
-        // Initialize Customer.io push messaging allows
-        // the SDK to automatically send FCM push tokens to
-        // Customer.io!
         MessagingPushFCM.initialize { config in
             config.autoFetchDeviceToken = true
         }
+
+        // Manually get FCM device token. Swizzling hasn't been working for me.
+        Messaging.messaging().delegate = self
+
+        // Register a 2nd push click listener besides the Customer.io SDK.
+        // Tests that the SDK is able to handle push clicks when there are multiple click listeners.
+        UNUserNotificationCenter.current().delegate = self
```

Comment:
Is this final for `main`? If yes, can we improve comments by adding some explanation from customer's perspective.

Reply:
I believe this response [here](https://github.com/customerio/customerio-ios/pull/403#discussion_r1425853552) also answers this question?

Reply:
Yeah, sounds good